main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -3436,8 +3523,10 @@ object Resolver {
           case _ => Result.Err(ResolutionError.IllegalType(tpe, loc))
         }
 
-      // Case 3: Enum. Return an object type.
+      // Case 3: Enum / Struct. Return an object type.
       case _: UnkindedType.Enum => Result.Ok(Class.forName("java.lang.Object"))
+      // TODO: Should this be a specific class instead
```

Comment:
I encourage you to use a code name for your TODOs. This makes it easier to find TODOs related to your project.

E.g. `TODO STRUCTS`